apps/pm-user/src/components/atoms/logo/Logo.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import styles from './logo.module.scss';
+
+const Logo = () => {
```

Comment:
logo component should have prop for the class to resize the logo

Reply:
In Desktop view: - We only need Logo in header. So, I didn't feel the need to make the Logo "resizeable".

For mobile-screens:- It would need writing a media query in CSS. When we reach that point, a smaller size for Logo's font can be specified directly.

Thus both cases are covered and there is no need to make Logo re-sizeable by passing a prop. Hence I didn't do it.